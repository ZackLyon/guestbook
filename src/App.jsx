import { UserProvider } from './context/UserContext.jsx';
import Header from './components/Header/Header';
import Footer from './components/Footer';
import Home from './views/Home';
import './App.css';

export default function App() {
  return (
    <div className="app-container">
      <UserProvider>
        <Header />
        <Home />
        <Footer />
      </UserProvider>
    </div>
  );
}
