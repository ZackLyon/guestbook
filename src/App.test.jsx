import { render } from '@testing-library/react';
import App from './App';
import { UserProvider } from './context/UserContext.jsx';

it('should display App view on the screen', () => {
  const { container } = render(
    <UserProvider>
      <App />
    </UserProvider>
  );

  expect(container).toMatchSnapshot();
});
