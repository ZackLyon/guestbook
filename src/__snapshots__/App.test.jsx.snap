// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should display App view on the screen 1`] = `
<div>
  <div
    class="app-container"
  >
    <header>
      <h1>
        Guestbook
      </h1>
      <h3>
        Welcome, Guest
      </h3>
    </header>
    <div>
      <section
        class="main-section"
      >
        <div
          class="inputContainer"
        >
          <form>
            <label>
              <input
                placeholder="name"
                type="text"
              />
            </label>
            <input
              placeholder="message"
              type="textarea"
              value=""
            />
            <button
              type="submit"
            >
              Submit
            </button>
          </form>
        </div>
        <div
          class="commentContainer"
        />
      </section>
    </div>
    <div />
  </div>
</div>
`;
